app/next.config.js

```diff
@@ -0,0 +1,14 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  i18n: {
+    // TODO: implement i18n internationalization-- look into nextJS subpath routing
+    locales: ['en-US', 'es-ES'],
+    defaultLocale: 'en-US',
+    localeSubpaths: {
+      es: "es"
+    },
+  },
+  reactStrictMode: true
```

Comment:
What does this do?

Reply:
NextJS documentation recommended adding this when I was reading up on linters. Here's a [link to the docs](https://reactjs.org/docs/strict-mode.html). React strict mode is a tool that provides additional warnings/error messages whenever React guidelines and recommended practices aren't being followed-- e.g. it identifies unsafe lifecyles. The warnings/errors only get logged when in developer mode so it wouldn't affect production.